src/notification/index.js

```diff
@@ -77,6 +78,10 @@ export const getAccountFromNotificationData = (
       realm_uri,
       parsed_url: realmUrl,
       match_count: urlMatches.length,
+      known_urls: knownUrls,
+      uniqueIdentities: identities
```

Comment:
nits:
* use snake_case to match the other keys -- at least the ones in this same warning-event blob :slightly_smiling_face: 

* name should say "count", like `match_count` does, to indicate it's the number of them and not a list of them